strapi/strapi
# chore: make vite the default bundler for the admin

### What does it do?

* Makes Vite the default bundler for build & develop of the admin panel
* reorganises the contributor docs

### Why is it needed?

* VITE VITE VITE

### How to test it?

* run `yarn develop --watch-admin`
* it should use vite (you can tell because it will be super fast)

### Related issue(s)/PR(s)

* resolves CONTENT-1636
